test/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/TimestampUtilsTest.java

```diff
@@ -15,6 +15,7 @@
  */
 package org.jitsi.impl.neomedia.rtp.remotebitrateestimator;
 
+import org.jitsi.impl.neomedia.rtp.TimestampUtils;
```

Comment:
star import (you can setup your ide with this https://gist.github.com/gpolitis/19b75e4814638f8153ab2369a8599549)